src/order/rel_classes.lean

```diff
@@ -19,6 +18,9 @@ universes u v
 
 variables {α : Type u} {β : Type v} {r : α → α → Prop} {s : β → β → Prop}
 
+@[elab_simple]
+lemma antisymm_of (r : α → α → Prop) [is_antisymm α r] {a b : α} : r a b → r b a → a = b := antisymm
```

Comment:
Wasn't this present in another PR with a comment? Do they depend on each other?

Reply:
No, they just both need the same lemma. I'll copy over the comment and make sure it's in the exact same place.